service/service.go

```diff
@@ -0,0 +1,58 @@
+package service
+
+import (
+	"encoding/json"
+	"github.com/Jeffail/gabs"
+	"github.com/blockassets/cgminer_client"
+	"io"
+	"io/ioutil"
+	"log"
+	"os/exec"
+	"time"
+)
+
+//TODO: refactor this file to separate out the service level commands into their own files
```

Comment:
Let's just do the simple TODO's right away.

Reply:
I had started to do that and then undid it as it actually needed more thought....

Reply:
damn interaction between gitfmt and intelij :-(